front/src/views/news/news.view.html

```diff
@@ -1,3 +1,25 @@
-<div>    
-  This is a news page.
+<div class = "news-component">    
+  <el-row v-for="newsItem in newsList" class = "row" :key = "newsItem.Id">
+    <el-col :span="9">
+      <el-image fit ='cover' :src = 'newsItem.previewImage | imageFullPath'></el-image>
+    </el-col>
+      <el-col :span="15">
+        <div class = "main-block">
+          <div class = "title">
+            <h1>{{ newsItem.title }}</h1>
+          </div>
+          <div class = "main-information">
+            <div class = "info-element secondary"><p>{{ newsItem.publishTime | convertDateTime('dd.MM.yyyy kk:mm') }}</p></div>
+            <div class = "info-element secondary"><p>{{ newsItem.countViews }} просмотров</p></div>
+            <div class = "secondary"><p>{{ newsItem.author.firstName }} {{ newsItem.author.lastName }}, </p></div>
+            <div class = "secondary"><p>{{ newsItem.categories | mapByProperty("name") | listStringify }}</p></div>
```

Comment:
i don't see in design categories

Reply:
it could be good to name to show meaning - in imperial style
.publish-date,
.views-count
.author

then to make margin
```
.publish-date,
.views-count {
  margin-left: 15px;
}
```

Reply:
actually, it could be better to make fixed size for this blocks - in case when count of views will have different digits (100 and 34 views) - layout should have same columns - or use el-row and el-coll

Reply:
there are categories after an author`s name in the design

Reply:
added el-row and el-col